app/views/declaration/previousDocuments/previous_documents.scala.html

```diff
@@ -162,14 +109,14 @@
 }
 
 @govukLayout(
-    title = Title("declaration.previousDocuments.title", "supplementary.consignmentReferences.heading"),
+    title = Title("declaration.previousDocuments.title", "declaration.previousDocuments.heading"),
```

Comment:
Does that work?  It looks like Navigator no longer has entries for "Document"

Reply:
Answered in separate comment